pkg/sql/rename_table.go

```diff
@@ -232,6 +233,29 @@ func (n *renameTableNode) startExec(params runParams) error {
 		return err
 	}
 
+	// If this table has row level ttl enabled, update the schedule_name of all
+	// row-level-ttl jobs with the name table name.
+	if tableDesc.HasRowLevelTTL() {
+		const (
+			opname = `update ttl job names`
+			query  = `update system.scheduled_jobs set schedule_name = 'row-level-ttl-' || $1 WHERE schedule_id = $2`
```

Comment:
nit: is it better to use captials and some parenthesis here like
```
UPDATE system.scheduled_jobs SET schedule_name = ('row-level-ttl-' || $1) WHERE schedule_id = $2
```

Reply:
Done.